site/learn/BestPractice-Caching.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Caching
+layout: ../_core/DocsLayout
+category: Best Practices
+permalink: /learn/caching/
+---
+
+> Providing Object Identifiers allows clients to build rich caches
```

Comment:
I disagree that this is associated with whether or not it is possible to build a rich cache, since it's definitely achievable without one. So I think this is a best practice, but not because it's easier to cache - it just makes your API cleaner since you have a consistent source of IDs for all objects.